From ee74ec4da045f764a0c2894a3b204e3e2b3a3ac2 Mon Sep 17 00:00:00 2001
From: JPyke3 <pyke.jacob1@gmail.com>
Date: Fri, 2 Oct 2020 01:22:23 +0200
Subject: [PATCH] Modified Grub to install using no-varam, this prevents the
 installer from touching EFI variables that the T2 Enabled MacBooks cannot
 have touched

Signed-off-by: JPyke3 <pyke.jacob1@gmail.com>
---
 src/modules/bootloader/bootloader.conf | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/modules/bootloader/bootloader.conf b/src/modules/bootloader/bootloader.conf
index 51337de9d..1bc993f73 100644
--- a/src/modules/bootloader/bootloader.conf
+++ b/src/modules/bootloader/bootloader.conf
@@ -26,7 +26,7 @@ fallbackKernelLine: ", with _manjaro_kernel_ (fallback initramfs)"
 # GRUB functionality (notably grub-probe) to work. As needed, you may use
 # complete paths like `/usr/bin/efibootmgr` for the executables.
 #
-grubInstall: "grub-install"
+grubInstall: "grub-install --target=x86_64-efi --efi-directory=/boot --bootloader-id=GRUB --no-nvram"
 grubMkconfig: "grub-mkconfig"
 grubCfg: "/boot/grub/grub.cfg"
 grubProbe: "grub-probe"
-- 
2.28.0

